Backpack-SwiftUI/CardCarousel/Classes/BPKCardCarousel.swift

```diff
@@ -66,23 +68,58 @@ internal struct InternalCardCarousel<Content: View>: View {
         content: [Content],
         currentIndex: Binding<Int>
     ) {
+        self.isIPad = UIDevice.current.name.hasPrefix("iPad")
+        self.size = size
         self._currentIndex = currentIndex
-        self.cardWidth = size.width * 0.8
+        self.cardWidth = isIPad ? (size.width * 0.8) * 0.4 : size.width * 0.8
         self.cardCount = content.count
         self._currentInternalIndex = State(initialValue: cardCount + 1 + currentIndex.wrappedValue)
-        self.content = content + content
+        
+        if isIPad && UIDevice.current.orientation.isLandscape {
+            self.content = content
+        } else {
+            self.content = content + content
+        }
+        
         focusOnCard = .init(currentInternalIndex)
     }
     
     var body: some View {
+        if isIPad && UIDevice.current.orientation.isLandscape {
```

Comment:
There is quite a jarring image animation when the iPad rotates, is there anything we can do to prevent that from happening? 

https://github.com/user-attachments/assets/39ed0ff6-d392-4b61-8114-c2fcfc8648d0